app/views/citizens/other_assets/show.html.erb

```diff
@@ -0,0 +1,42 @@
+
+<%= render partial: 'shared/forms/error_summary', locals: { model: @form } %>
```

Comment:
This page is missing the back link and the title, and the indentation is not 100% correct.
Also, FYI, from talking with Simon, The form and its title should be included in a "two-third" div:
```
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
```
However, the error summary should be full width (above these divs)